week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -22,57 +22,86 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 //1)Create the function inside, Think about nested loops
 function sortAges(arr) {
   //create function here
-}
-
-/* ======= TESTS - DO NOT MODIFY ===== */
-
-const agesCase1 = [
-  "🎹",
-  100,
-  "💩",
-  55,
-  "🥵",
-  "🙈",
-  45,
-  "🍕",
-  "Sanyia",
-  66,
-  "James",
-  23,
-  "🎖",
-  "Ismeal",
-];
-const agesCase2 = ["28", 100, 60, 55, "75", "🍕", "Elamin"];
-
-function arraysEqual(a, b) {
-  if (a === b) return true;
-  if (a == null || b == null) return false;
-  if (a.length != b.length) return false;
-
-  for (let i = 0; i < a.length; ++i) {
-    if (a[i] !== b[i]) return false;
+ //node removes any non-number element from the array
+  var sortedResult=arr.filter((item)=>
+  {
+  return(typeof item ==='number')
+  });
+   //bubble sort
+   //code is from https://khan4019.github.io/front-end-Interview-Questions/sort.html.
+
+   
+   var len = arr.length;

```

Comment:
Hi Tas, did you research this sort function online ?
Are you happy with everything that it is doing ?